scrapper/orchestrator.py

```diff
@@ -169,15 +167,15 @@ def scrapeAndInsertData( p_games, p_batch, p_con ):
 start_date = args.startDate
 end_date = args.endDate
 batch = args.batch
-sport_names = lookupByValue(sport_id_map, args.lg)
-league_names = lookupByValue(league_id_map, args.lg)
-team_names = lookupByValue(team_id_map, args.teams)
+sport_names = [lookupByValue(sport_id_map,league) for league in args.lg.split(",")]
+league_names = [lookupByValue(league_id_map,league) for league in args.lg.split(",")]
+team_names = [lookupByValue(team_id_map, team) for team in args.teams.split(",")] if args.teams else []
```

Comment:
Again, not sure about this one. Can we check if the APi will limit the number of records returned?